stdlib/tools/swift-reflection-test/swift-reflection-test.c

```diff
@@ -219,6 +204,10 @@ const void *PipeMemoryReader_readBytes(void *Context, swift_addr_t Address,
   write(WriteFD, &TargetSize, sizeof(size_t));
   
   void *Buf = malloc(Size);
+
+  if (Buf == NULL) {
+    errnoAndExit("malloc failed");
+  }
```

Comment:
We generally assume the result of malloc is valid: if it's not, we'll crash soon enough anyway, and that's fine.